addons/web/static/src/core/record_selectors/record_selector.xml

```diff
@@ -2,18 +2,21 @@
 <templates xml:space="preserve">
 
     <t t-name="web.RecordSelector" >
-        <RecordAutocomplete
-            resModel="props.resModel"
-            value="displayName"
-            domain="props.domain"
-            context="props.context"
-            className="'h-100 o_record_selector'"
-            fieldString="props.fieldString"
-            placeholder="props.placeholder"
-            multiSelect="false"
-            getIds="() => []"
-            update.bind="update"
-        />
+        <div class="o_input d-flex flex-wrap gap-1 o_record_selector">
```

Comment:
via AutoComplete, the class o_input is already set. Do we need it here? Same question for web.MultiRecordSelector

Reply:
Hello

Both the `o_input` are necessary. If the parent `o_input` is removed then the `underline` or `border-bottom` would not be visible (below the caret) when hovered over selectors. 
https://github.com/odoo/odoo/blob/17.0/addons/web/static/src/views/fields/fields.scss#L50-L54

If the child `o_input` is removed then the record autocomplete would have visible borders around it.
https://github.com/odoo/odoo/blob/17.0/addons/web/static/src/views/fields/fields.scss#L57-L59